osu.Framework/Graphics/Sprites/SpriteText.cs

```diff
@@ -636,6 +636,6 @@ public float LineBaseHeight
             }
         }
 
-        public IEnumerable<string> FilterTerms => displayedText.Yield();
+        public IEnumerable<LocalisableString> FilterTerms => new LocalisableString(displayedText).Yield();
```

Comment:
Why use `displayedText` and not

```suggestion
        public IEnumerable<LocalisableString> FilterTerms => Text.Yield();
```